src/view/components/sandboxSelector.jsx

```diff
@@ -29,18 +29,22 @@ const getLabel = sandbox => {
 const SandboxSelector = ({
   defaultSelectedSandbox,
   onSelectionChange,
-  errorMessage,
-  initInfo
+  initInfo,
+  items,
```

Comment:
How about we rename these 2 options to what they actually are:

`items` -> `sandboxes`
`otherProps` -> `sandboxProps`